src/backend/api/Fusion.Resources.Api/Controllers/Departments/DepartmentsController.cs

```diff
@@ -15,16 +16,17 @@ namespace Fusion.Resources.Api.Controllers.Departments
     [ApiController]
     public class DepartmentsController : ResourceControllerBase
     {
+        private readonly ResourcesDbContext db;
```

Comment:
Don't think you should introduce db context here.
Should use query handler instead.